package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "gulp-livingstyleguide",
-  "version": "1.0.0",
+  "name": "gulp-livingstyleguide2",
+  "version": "1.0.1",
```

Comment:
As this is more a feature (new option) than a bug fix I would suggest `1.1.0` instead of `1.0.1`.